wfl/Models/Restaurant.cs

```diff
@@ -19,5 +21,22 @@ public bool CanHaveLinkToMenu()
             }
             return false;
         }
+        public static bool CanConstructFromDto(RestaurantDto dto)
+        {
+            if (dto?.Name == null || (dto.MenuLink != null && !Uri.IsWellFormedUriString(dto.MenuLink.ToString(), UriKind.Absolute)))
+            {
+                return false;
+            }
+            return true;
+        }
+        public static async Task<bool> CanUpdateRestaurantAsync(RestaurantDto dto, string id, IDBManager<Restaurant> restaurantDBManager)
```

Comment:
Don't make this method static.  Simply use the actual Restaurant object and then check if it can perform the update.  Then we can remove the dbmanager requirement from here as well.